templates/eap-xp1-basic-s2i.json

```diff
@@ -0,0 +1,539 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,eap-xp,javaee,java,jboss",
+            "version": "1.0",
+            "openshift.io/display-name": "JBoss EAP XP 1.0",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example JBoss Enterprise Application Platform application. For more information about using this template, see https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/eap-xp1/README.adoc/README.adoc",
```

Comment:
"Example of an application based on JBoss EAP XP"

Reply:
URL should be https://github.com/jboss-container-images/jboss-eap-openshift-templates/blob/eap-xp1/README.adoc